### ISSUE TITLE
MathpixPDFLoader doesn't work. 

### ISSUE THREAD

**Issue by Monketoo** [langchain-ai/langchain#7282]
### System Info

Getting ''' ValueError: Unable to send PDF to Mathpix'''  while using MathpixPDFLoader. 

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [X] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

math_pix_loaded = MathpixPDFLoader(file_path)
load_list_mathpix = math_pix_loaded.load()

### Expected behavior

A list of pages to be returned.

**Comment by MatousEibich** [comment]
Hello, same issue, would appreciate some clarification, already paid MathPix sign up fee.

**Comment by jreinjr** [comment]
Hi! There's a bug in the MathpixPDFLoader class - in the __init__ function, the processed_file_format variable is set to "mmd" by default. Change it to "md" and it works properly!

**Comment by sonnyjon** [comment]
Hi,

I have the same issue. Would someone please implement jrinjr's fix in the next release so that we don't have to maintain our own version of the MathpixPDFLoader class?

**Comment by efriis** [comment]
Should be fixed in 0.0.305! Thanks @jreinjr for fixing

**Comment by sonnyjon** [comment]
Something changed between v0.0.300 and v0.0.305 in pdf.py that is causing a TimeoutError:

```
  File "C:\Workspace\python\langchain\demo-projects\pdf-loaders\venv\lib\site-packages\langchain\document_loaders\pdf.py", line 435, in load
    contents = self.get_processed_pdf(pdf_id)
  File "C:\Workspace\python\langchain\demo-projects\pdf-loaders\venv\lib\site-packages\langchain\document_loaders\pdf.py", line 405, in get_processed_pdf
    self.wait_for_processing(pdf_id)
  File "C:\Workspace\python\langchain\demo-projects\pdf-loaders\venv\lib\site-packages\langchain\document_loaders\pdf.py", line 402, in wait_for_processing
    raise TimeoutError
TimeoutError
```

I messaged Mathpix support and they indicated that nothing changed on their end:
 
> Hello, yes I've just checked and those two files are getting converted fine on our side
> This is most likely caused on LangChain side.. but without any logs it's difficult to say what can be wrong there
> Please keep us updated as well, once there will be news from LangChain
>  
> Thanks!

Is this a known issue or do I need to open an issue for this? Thanks.
